docs/01-membership.md

```diff
@@ -59,6 +59,7 @@ Individuals from active working groups produce the membership by opting into Pro
 | **Statelessness** (2 contributors) | | |
```

Comment:
This would be 3 according to the worldview of this PR for now -- since the other one has not been merged yet

Reply:
to be clear, #400 changes that very same line, I'm trying to avoid conflicts as much as possible.